Manual/Releases/v4.15.0.lean

```diff
@@ -0,0 +1,665 @@
+/-
+Copyright (c) 2025 Lean FRO LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Joachim Breitner
+-/
+
+import VersoManual
+
+import Manual.Meta.Markdown
+
+open Manual
+open Verso.Genre
+
+
+#doc (Manual) "v4.15.0" =>
+%%%
+tag := "release-v4.15.0"
+file := "v4.15.0"
+%%%
+
```

Comment:
In the original context, these pages have dates associated with them. What about adding the release date here?